### ISSUE TITLE
Error on Loading `emqttd_auth_http`

### ISSUE THREAD

**Issue by drasko** [emqtt/emqttd#691]
Hello,
I am trying to load `emqtt_auth_http` and I am seeing this error:

```
drasko@Lenin:~/emqtt/emqttd$ ./bin/emqttd-2.0 start
drasko@Lenin:~/emqtt/emqttd$ ./bin/emqttd_ctl plugins load emqttd_auth_http
load plugin error: {emqttd_auth_http,
                    {bad_return,
                     {{emqttd_auth_http_app,start,[normal,[]]},
                      {'EXIT',
                       {function_clause,
                        [{emqttd_auth_http_app,record,
                          [[{method,post},
                            {url,"http://localhost:8080/mqtt/acl"},
                            {params,
                             [{access,"%A"},
                              {username,"%u"},
                              {clientid,"%c"},
                              {ipaddr,"%a"},
                              {topic,"%t"}]}]],
                          [{file,"src/emqttd_auth_http_app.erl"},{line,72}]},
                         {emqttd_auth_http_app,'-register_acl_mod/1-fun-0-',
                          2,
                          [{file,"src/emqttd_auth_http_app.erl"},{line,50}]},
                         {emqttd_auth_http_app,start,2,
                          [{file,"src/emqttd_auth_http_app.erl"},{line,41}]},
                         {application_master,start_it_old,4,
                          [{file,"application_master.erl"},{line,273}]}]}}}}}
drasko@Lenin:~/emqtt/emqttd$
```

Trying `unload` just after gives this:
```
drasko@Lenin:~/emqtt/emqttd$ ./bin/emqttd_ctl plugins unload emqttd_auth_http
unload plugin error: not_started
```

However, it looks like plugin was partially loaded, as I am getting this:
```
drasko@Lenin:~/go/src/github.com/mainflux/mainflux-lite$ mosquitto_sub -h localhost -t mainflux/66be841c-8e9e-4620-b0b6-7c4b5ed823eb
Connection Refused: bad user name or password.
Connection Refused: bad user name or password.
...
```

Am I missing something in the configuration or the plugin is broken?

**Comment by drasko** [comment]
Ah, I just saw that `emqttd_auth_http` plugin is supported only in 1.1 release.

Will it be supported in 2.0?

**Comment by huangdan** [comment]
@drasko The issue has been fixed in V2.0_bete.2 ;Please upgrade

**Comment by emqplus** [comment]
Hi @drasko, the issue has been fixed in 2.0-beta.3
